app/commands/build_claim_command.rb

```diff
@@ -17,20 +17,30 @@ class BuildClaimCommand < BaseCommand
   attribute :miscellaneous_information, :string
   attribute :employment_details
   attribute :is_unfair_dismissal, :boolean
+  attribute :pdf_template_reference, :string, default: 'et1-v1-en'
 
-  def initialize(*)
-    super
-    self.reference_service = ReferenceService
+  validates :pdf_template_reference, inclusion: { in: ['et1-v1-en', 'et1-v1-cy'] }
+
+  def initialize(*args, reference_service: ReferenceService, allocator_service: UploadedFileAllocatorService.new, **kw_args)
+    super(*args, **kw_args)
+    self.reference_service = reference_service
+    self.allocator_service = allocator_service
   end
 
   def apply(root_object, meta: {})
     apply_root_attributes(attributes, to: root_object)
-    meta.merge! reference: root_object.reference
+    allocate_pdf_file(root_object)
+    meta.merge! reference: root_object.reference,
+                pdf_url: allocator_service.allocated_url
```

Comment:
As in ET3 - ET1 will need to know where the pdf file will be even before it is generated. This allows for the link to be displayed with some client side polling logic